### PR TITLE
#2 Initialize with Spring WebFlux

### PR DESCRIPTION
**Pull request by bagjinok**
Initialize with Spring WebFlux

**CHANGES**
- Add Gradle settings
- Add webflux and lombok library
- Implement test `YourLinkRouter` and `YourLinkHandler` using functional programming
    - https://docs.spring.io/spring/docs/current/spring-framework-reference/web-reactive.html#webflux-fn

you can test with below curl
```
curl -X GET \
   http://127.0.0.1:8080/your/links \
   -H 'Cache-Control: no-cache' \
   -H 'Content-Type: application/json'
```

### CODE REVIEW

File: api/src/main/java/com/bringback/yourlink/api/model/YourLink.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.bringback.yourlink.api.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class YourLink {
+    private String url;
+    private boolean isRed;
```
Comment (echo-windingroad) [inline api/src/main/java/com/bringback/yourlink/api/model/YourLink.java]: **isRed** variable means **isRead**?

Comment (windingroadss) [reply api/src/main/java/com/bringback/yourlink/api/model/YourLink.java]: Review notification test

Comment (windingroadss) [reply api/src/main/java/com/bringback/yourlink/api/model/YourLink.java]: Review notification test2

Comment (bagjinok) [reply api/src/main/java/com/bringback/yourlink/api/model/YourLink.java]: Oh :-) I will change to `isRead`.
```suggestion
    private boolean isRead;
```

Comment (bagjinok) [reply api/src/main/java/com/bringback/yourlink/api/model/YourLink.java]: Oh :-) I will change to `read`
```suggestion
    private boolean read;
```

**Pull request merged**